### PR TITLE
Sim 2 landing, base line regression and data display

### PR DESCRIPTION
**Pull request by Evan-CTL**

### CODE REVIEW

File: media/js/src/multipleChoiceQuestion.jsx [hunk 1]
```diff
@@ -41,9 +41,7 @@ export const MultipleChoiceQuestion = ({
                 </header>
                 <div className="simulation__step-content">
                     <p>
-                        You have conpleted all the hypothesis tests in this
-                        section. Based on what you&rsquo;ve learned from this
-                        exercise, which of the following is true?
+                        {question}
```
Comment (ndittren) [inline media/js/src/multipleChoiceQuestion.jsx:44]: This change is already in the code. It's weird that the old code is still here. Can you try rebasing with the new main?

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "eslint-plugin-cypress": "^3.0.3",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-security": "~3.0.0",
+    "json-loader": "^0.5.7",
```
Comment (ndittren) [inline package.json:35]: Not seeing this used anywhere so I think we can remove

File: package.json [hunk 3]
```diff
@@ -47,6 +48,7 @@
     "axios": "^1.6.5",
     "bootstrap": "^5.3.2",
     "file-saver": "^2.0.5",
+    "json-loader": "^0.5.7",
```
Comment (ndittren) [inline package.json:51]: Same here, we can remove

**Review by ndittren — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Evan-CTL) [reply package.json]: It was to handle the json data, but it ended up not getting used

File: package.json (hunk 3 above)
Comment (Evan-CTL) [reply package.json]: Same here

File: media/js/src/multipleChoiceQuestion.jsx (hunk 1 above)
Comment (Evan-CTL) [reply media/js/src/multipleChoiceQuestion.jsx]: Rebase done

**Review by ndittren — APPROVED** [review]
Looks good!

**Pull request merged**